### ISSUE TITLE
Add vim docs

### ISSUE THREAD

**Issue by oryband** [Chiel92/vim-autoformat#120]
They're missing. `:help autoformat` returns built-in docs.

**Comment by Chiel92** [comment]
I don't think we should overwrite that vim-docs entry, as it still provides valuable information. Vim-autoformat does not obsolete the builtin functionality, it kind of extends it.

That being said, I wouldn't hurt to have our own vim-docs entry. We could use https://github.com/xolox/vim-tools to export our README.md to vim-docs format. I would be open to a PR implementing this. The README should get a paragraph with instructions about this though, since with every contribution to the README, the vim-docs would have to be updated as well.
